### ISSUE TITLE
First and last slides are rendered extra

### ISSUE THREAD

**Issue by masamichhhhi** [femioladeji/react-slideshow#148]
I want to define each slide as a separate component.
I have defined the Image component as follows, but when I check it using the react development tools, it looks like two extra components are being rendered.
```tsx
import { Slide } from "react-slideshow-image";
import "react-slideshow-image/dist/styles.css";

const slideImages = [
  {
    url: "images/slide_2.jpg",
    caption: "Slide 1",
  },
  {
    url: "images/slide_3.jpg",
    caption: "Slide 2",
  },
  {
    url: "images/slide_4.jpg",
    caption: "Slide 3",
  },
];

const Component = (props: { imageUrl: string }) => {
  return (
    <div className="each-slide" style={{ margin: "50px" }}>
      <div>{props.imageUrl}</div>
    </div>
  );
};

const Slideshow = () => {
  return (
    <div className="slide-container">
      <Slide infinite={false}>
        {slideImages.map((slideImage, index) => (
          <Component key={index} imageUrl={slideImage.url} />
        ))}
      </Slide>
    </div>
  );
};

export default Slideshow;

```
<img width="1800" alt="Screen Shot 2021-12-21 at 23 56 01" src="https://user-images.githubusercontent.com/43578455/146950532-12abe837-f948-452a-9a89-22268144f7ac.png">

It looks like the first and last slides are being rendered in duplicate, is there any way to get rid of this?

**Comment by femioladeji** [comment]
@masamichhhhi yeah they are intentional. They are added to ensure a smooth experience when you're sliding from one slide to another. For example, if you are on the last slide and you continue to swipe, the expectation is that the first slide shows. That's why those two slides are added. You can check this link out [https://codesandbox.io/s/ancient-microservice-l3red?file=/src/slideshow.js]https://codesandbox.io/s/ancient-microservice-l3red?file=/src/slideshow.js

**Comment by masamichhhhi** [comment]
I see. but,  I think this way should be applied only when `infinite=true`. Could it be disabled when `infinite=false`?

**Comment by femioladeji** [comment]
@masamichhhhi yeah but also the `infinite` prop can be changed.
Just curious; are you having some issues because of the addition of those extra slides

**Comment by masamichhhhi** [comment]
@femioladeji 
yes, I'm using `<video>` tag in Slide component and want to play and pause videos from an external button.
so, I've tried to get video element by `document.getElementsByClassName` and play video like following.
```ts
const videoElement = document.getElementsByClassName('video 1')[0]
videoElement.play()
```
but I got two same element and video doen't play.

In this use case, shouldn't I use this library?

**Comment by femioladeji** [comment]
@masamichhhhi gotcha. I've raised a PR to fix this. I'll let you know once the new version goes live

**Comment by masamichhhhi** [comment]
Thank you for dealing with it so quickly!

**Comment by femioladeji** [comment]
@masamichhhhi sorry for the delay, a new version has been released v3.6.1. That should work for you

**Comment by vinidisanti** [comment]
Hey there!
It's not working again. It only works if slidesToShow and slidesToScroll are the same.
